hack/api-description-whitelist.txt

```diff
@@ -0,0 +1,329 @@
+## ---------------------------------------------- ##\
+## THIS WHITELIST ALLOWS FOR THE VERIFY SCRIPT TO ##\
+## SUCCEED WHILE DESCRIPTIONS ARE ADDED. IF A NEW ##\
+## DESCRIPTION IS ADDED, THE CORRESPONDING ENTRY  ##\
+## IN THIS LIST SHOULD BE REMOVED.                ##\
+## ADDING NEW ENTRIES TO THIS LIST IS DISALLOWED  ##\
+## ---------------------------------------------- ##
+api.patch
+integer
+json.watchevent
+patch.object
+runtime.rawextension
+v1.awselasticblockstorevolumesource
+v1beta3.awselasticblockstorevolumesource
```

Comment:
Any idea how or why you're picking up v1beta3?  We don't have a generated swagger spec for that one